apps/ledger-live-mobile/src/screens/OperationDetails/Extra.tsx

```diff
@@ -1,17 +1,24 @@
 import React from "react";
 import { useTranslation } from "react-i18next";
+import { Operation } from "@ledgerhq/types-live";
 import Section from "./Section";
 
 type Props = {
-  extra: Record<string, string>;
+  operation: Operation;
 };
-export default function OperationDetailsExtra({ extra }: Props) {
+export default function OperationDetailsExtra({ operation }: Props) {
   const { t } = useTranslation();
-  return (
+
+  // Safety type checks
+  return operation.extra &&
+    typeof operation.extra === "object" &&
+    !Array.isArray(operation.extra) ? (
```

Comment:
```suggestion
```

I don't think this check is needed as it's `Record<string, any>`

Reply:
It's not anymore `Record<string, any>` but `unknown` instead.